src/app/campaign-list/campaign-list.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Campaign, CampaignListService } from './campaign-list.service';
+
+@Component({
+  selector: 'grove-campaign-list',
+  template: `
+    <ul>
+      <li *ngFor="let campaign of campaigns$ | async">
+        <grove-campaign-card [campaign]="campaign"></grove-campaign-card>
+      </li>
+      <ng-container *ngIf="loading">
+        <li *ngFor="let nothing of loadingCards(loadingCount); let i = index">
+          <grove-campaign-card></grove-campaign-card>
+        </li>
+      </ng-container>
+    </ul>
+    <prx-paging
+      [currentPage]="currentPage"
+      [totalPages]="totalPages"
+      (showPage)="showPage($event)">
+    </prx-paging>
+  `,
+  styleUrls: ['campaign-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
`changeDetection: ChangeDetectionStrategy.OnPush` the `| async` pipe marks a component for change detection